app/src/lang/translations/pt-BR.yaml

```diff
@@ -19,6 +19,9 @@
 #'Int32Array', 'Uint32Array', 'Float32Array', 'Float64Array', 'Map', 'Set', 'WeakMap', 'WeakSet',
 #'SIMD', 'ArrayBuffer', 'DataView', 'JSON', 'Promise', 'Generator', 'GeneratorFunction', 'Reflect',
 #'Proxy', 'Intl'
+Published: Publicado
+Draft: Rascunho
+Archived: Arquivado
```

Comment:
Lets make these all lowercase to be consistent with the rest of the translations